design/ceph/ceph-exporter.md

```diff
@@ -0,0 +1,83 @@
+# Ceph exporter
+
+### Goals
+
+Use Ceph exporter as the new source of metrics based on performance counters coming from every Ceph daemon.
+
+## Use case
+
+Ceph exporter was created to avoid performance problems at scale obtaining metrics in clusters with high number of daemons (mainly OSDs).
+The Prometheus manager module was not enough performant to obtain the performance counters from all the daemons along the whole Ceph cluster, so the solution for avoid this problem in big clusters is to have a new daemon collecting all the performance counters from all the daemons in each host, and expose them as standard prometheus metrics at /metrics endpoint for prometheus to scrape each of those endpoints.
+
+### Why do we need this daemon in Rook
+
+With the Ceph exporter, the source of metrics changes in the Ceph clusters.
+There will be two main sources of metrics:
+- Prometheus manager module: It is responsible for exposing all metrics other than ceph daemons performance counters.
+- Ceph exporter: It is responsible for exposing only ceph daemons performance counters as prometheus metrics.
+
+Ceph exporter will be deployed by default in rook ceph cluster. But in case if pod fails to start or some issues with starting the container, it is still possible to fetch perf counters metrics from prometheus module by setting module option `fetch_perf_counters_metrics` to true.\
```

Comment:
If you want to mention it's still possible to enable the module setting, let's mention it separately to be clear that rook will not enable it, and we don't see the need for it since we can behave differently depending on the ceph version.
```suggestion
Ceph exporter will be deployed in Rook when the `monitoring.enabled: true` setting is applied in the CephCluster CR, and the Ceph version is at least 17.2.x **(insert expected version)**`
```

Reply:
Ok makes sense! and by separately you mean somewhere in rook ceph documentation right?